src/System.Globalization/tests/CharUnicodeInfo/CharUnicodeInfoTestData.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace System.Globalization.Tests
+{
+    public static class CharUnicodeInfoTestData
+    {
+        private static List<CharUnicodeInfoTestCase> s_testCases;
+        public static List<CharUnicodeInfoTestCase> TestCases
+        {
+            get
+            {
+                if (s_testCases == null)
+                {
+                    s_testCases = new List<CharUnicodeInfoTestCase>(); 
+                    string fileName = "UnicodeData.8.0.txt";
+                    Stream stream = typeof(CharUnicodeInfoGetUnicodeCategoryTests).GetTypeInfo().Assembly.GetManifestResourceStream(fileName);
```

Comment:
We just changed the name of some resources in https://github.com/dotnet/corefx/pull/7158 to workaround an msbuild issue.  You should kick off an outerloop OSX run on the PR to ensure this doesn't break things.